examples/search.js

```diff
@@ -38,7 +38,7 @@ const puppeteer = require('puppeteer');
   await page.click(allResultsSelector);
 
   // Wait for the results page to load and display the results.
-  const resultsSelector = '.gsc-results .gs-title';
+  const resultsSelector = '.gsc-table-result a.gs-title';
```

Comment:
```suggestion
  const resultsSelector = '.gsc-table-result a.gs-title[href]';
```
I ran it locally to verify and it seem it pick up a template html, so this insures that we don't get that in our query,